### PR TITLE
blog: docker disk space

### PR DESCRIPTION
**Pull request by maiamcc**
images tbd but wanted to get some eyes on the text

### CODE REVIEW

File: blog/_posts/2019-10-25-build-cache-spark-joy.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* Your project eats up a ton of space, and you want to blow away the maximum possible amount of stuff every time; set the max age really low. (Remember that no matter how low you set the max age, we'll only prune objects that are _not in use_. So, whatever images you're currently running are always safe.)
+```
+docker_pruner_settings(max_age_mins=15)
+```
+* The amount of space you use is unpredictable and doesn't correlate with time Tilt has been up for; instead of pruning every X hours, prune every Y builds instead.
+```
+docker_pruner_settings(num_builds=10)
+```
+
+We hope the Docker Pruner helps keep your disk usage in check, so you can stay in flow without worrying about finicky errors. Let us know how you're finding it! [Read more about the settings in the docs.](https://docs.tilt.dev/api.html#api.docker_prune_settings)) (Don’t like it? Disable it in your Tiltfile, and let us know what would make it better.)
```
Comment (jazzdan) [inline blog/_posts/2019-10-25-build-cache-spark-joy.md]:
```suggestion
We hope the Docker Pruner helps keep your disk usage in check, so you can stay in flow without worrying about finicky errors. Let us know how you're finding it! ([Read more about the settings in the docs.](https://docs.tilt.dev/api.html#api.docker_prune_settings)) (Don’t like it? Disable it in your Tiltfile, and let us know what would make it better.)
```

I think there are too many parentheticals here though.

File: blog/_posts/2019-10-25-build-cache-spark-joy.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+slug: build-cache-spark-joy
+date: 2019-10-16
+author: maia
+layout: blog
+title: "Does This Build Cache Spark Joy?"
+subtitle: "Pruning Away your Docker Disk Space Woes"
+image: "marie-kondo-containers.jpg"
+image_caption: "Do you really need all those stopped containers?"
```
Comment (nicks) [inline blog/_posts/2019-10-25-build-cache-spark-joy.md]: can you also credit the image?

File: blog/_posts/2019-10-25-build-cache-spark-joy.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+The more opaque form of this error is when your _local Kubernetes cluster_ (e.g. Kubernetes for Docker for Mac, or Minikube, if you’re also using the Minikube Docker instance) starts throwing “DiskPressure” errors. Recall that local k8s clusters are (generally) single-node clusters, i.e. all the k8s stuff happening on a single VM. Funnily, all your Docker storage is _also_ on that same VM; thus, if you have too much junk in your Docker storage, it takes away space that k8s would otherwise want to use, so k8s starts complaining about a lack of space. I won’t say that this is always the case, but often, “DiskPressure” errors on your local k8s cluster are, at their root, Docker disk space problems, and not k8s specific; try the steps below and see if the errors go away.
+
+### What's eating you(r disk space)?
+If you think you’re running out of Docker storage space, dig in with [`docker system df`](XXX) to see where your space is going. You’ll see stats for four types of
```
Comment (nicks) [inline blog/_posts/2019-10-25-build-cache-spark-joy.md]: XXX

Comment (maiamcc) [reply blog/_posts/2019-10-25-build-cache-spark-joy.md]: 😬ty good catch

File: blog/_posts/2019-10-25-build-cache-spark-joy.md (hunk 2 above)
Comment (maiamcc) [reply blog/_posts/2019-10-25-build-cache-spark-joy.md]: Done.

**Pull request merged**